Cargo.toml

```diff
@@ -30,7 +32,9 @@ bench = []
 
 [profile.release]
 debug = true
-lto = "thin"
+lto = true
```

Comment:
I'd prefer the optimization changes in a separate PR with benchmark results to justify them, since I'd expect this to increase compilation time, and IIRC when I last tried a couple years ago I couldn't find a significant performance benefit from full LTO or `codegen-units = 1`.

Reply:
Done in https://github.com/djudd/human-name/pull/7/commits/6afab67009c8ce55bc068b91223d17897fe85eab.

Will do a separate PR with the attached benchmarks this weekend.